ios.py

```diff
@@ -121,13 +121,7 @@ def setup_ios_device_template(env: dict, opts: iOSOpts, target: str):
     	'-DSMALL_CONFIG', '-D_XOPEN_SOURCE', '-DHOST_IOS', '-DHAVE_LARGE_FILE_SUPPORT=1'
     ]
 
-    LDFLAGS = []
-
-    # https://github.com/mono/mono/issues/19393
-    if os.environ.get('DISABLE_NO_WEAK_IMPORTS', '0') != '1':
-        LDFLAGS += ['-Wl,-no_weak_imports']
```

Comment:
I would have otherwise defaulted to disabling this since it doesn't build out of the box, but I figure we might as well always disable it.

Reply:
I think it's fine, considering they seem to be removing this in Mono as well: https://github.com/mono/mono/commit/0e3d75d6d6d6b6c404a6d02f22137262f0107d6b

Reply:
Looks like we could upgrade to `mono-6.12.0.111` and get this upstream fix: https://github.com/mono/mono/commit/124f1157141b323b6de883b45021170f1fe70539

Reply:
I bumped to `mono-6.12.0.111` so I dropped the other patch for Xcode 12 support too.

Reply:
> I bumped to mono-6.12.0.111 so I dropped the other patch for Xcode 12 support too. 

Well, looks like that upstream fix was not enough, the `sys-uio.c` patch is still needed:
https://github.com/godotengine/godot-mono-builds/pull/15/checks?check_run_id=1407141047